Project/structure_new.html

```diff
@@ -160,9 +167,19 @@ <h2 class="map-box__title">Awesome map</h2>
                 </ul>
             </nav>
         </div>
-    </footer>
+    </footer>    
+
     <script src="js/filter.js"></script>
+    
     <script src="js/menu.js"></script>
+    <script src="js/content.js"></script>
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCqvp81Gng_rK5OAITvVSxkJ9RhGQGKSrE"
```

Comment:
please use `async` *or* `defer`, not both - checkout [this](http://www.growingwiththeweb.com/2014/02/async-vs-defer-attributes.html) async/defer article